docs/repositories/issues.md

```diff
@@ -53,7 +53,9 @@ You can define one or more of the following filters:
 
 ## Ignoring and managing issues
 
-Use the options in the cogwheel menu of each issue to:
+Use the options in the options menu of each issue to:
```

Comment:
Good catch. I initially called it "three-dots menu" since it's not a cogwheel anymore, but the name looked weird and I found out it's also called "options menu". I'd still like to refer to the menu icon. How about:

```suggestion
Click the **options** menu of an issue to:
```

Reply:
> Click the **options** menu of an issue to:

We should use **bold** formatting for text that matches the text on the UI, which is not the case. 🤔 
On the other hand, if we instruct the user to "click" the options menu, we'll be missing the second click on the desired option. 😅 

After looking at some related content on the MS style guide (didn't find a match to our use case, I found [this one](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/m/menu-item), [this other one](https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui), and also [this one](https://learn.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions)), I don't have a good alternative. 😝 

So, if you want to use the "options menu", probably the best ends up being the original:

```suggestion
Use the options in the options menu of each issue to:
```

Reply:
Let's use your initial suggestion then:

```suggestion
Use the options in the menu of each issue to:
```